massmigration/backends/djangae.py

```diff
@@ -34,20 +34,21 @@ def run_simple(self, migration):
         defer(migration.wrapped_operation, _queue=self._get_queue_name(migration))
         logger.info("Deferred task to run single-task migration %s", migration.key)
 
-    def run_mapper(self, migration):
+    def run_mapper(self, migration, db_alias=None):
         # Use `defer_iteration_with_finalize` to do the processing with whichever key_ranges_getter
         # is appropriate for the DB.
-        queryset = migration.get_queryset()
+        queryset = migration.get_queryset(db_alias)
         key_ranges_getter = self._key_ranges_getter(queryset)
-        with get_transaction().atomic():
-            attempt_uuid = migration.mark_as_started()
+        with get_transaction().atomic(using=db_alias):
+            attempt_uuid = migration.mark_as_started(db_alias)
             defer_iteration_with_finalize(
                 queryset,
                 self._call_mapper_wrapped_operation,
                 self._mark_mapper_as_finished,
                 key_ranges_getter=key_ranges_getter,
                 migration=migration,
                 attempt_uuid=attempt_uuid,
+                db_alias=db_alias,
```

Comment:
`defer_iteration_with_finalize` has been tweaked/fixed to use `queryset.db` (see [here](https://gitlab.com/potato-oss/djangae/djangae/-/commit/35aede5fce374b19723f05ef6bfddfac63d1fbeb#c79213114fe1dc0665f603602e4946717f0ad62a_559_557)), so we don't need to explicitly set it. 

We're passing `db_alias` here so that is passed forward to the various methods (`_call_mapper_wrapped_operation` and `_mark_mapper_as_finished`).
Does it make sense?